IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgServiceElements/Types/IfcDistributionSystemEnum/DocEnumeration.xml

```diff
@@ -56,6 +56,7 @@
 		<DocConstant xsi:nil="true" href="MONITORINGSYSTEM_0WEnaOfI926w4GZGge3P" />
 		<DocConstant xsi:nil="true" href="USERDEFINED_2mLZb0gbX8OuHktnAtk7xk" />
 		<DocConstant xsi:nil="true" href="NOTDEFINED_2CUVQ2L_19c9FtUrDEPu4R" />
+		<DocConstant xsi:nil="true" href="SAFETY_1Pxj130uHFSPDGFuQSQy0c" />
```

Comment:
I guess this one could be moved above USERDEFINED and NOTDEFINED.

Reply:
Done.